Makefile

```diff
@@ -220,7 +220,7 @@ test-e2e: e2e-setup deploy-crds
 		operator-sdk --verbose test local ./test/e2e \
 			--namespace $(TEST_NAMESPACE) \
 			--up-local \
-			--go-test-flags "-timeout=15m" \
+			--go-test-flags "-timeout=108m" \
```

Comment:
Hi @pratikjagrut any specific reason to bump the timeout to 1.8 hrs from 15 mins?

Reply:
I've mentioned it in my commit message.

Reply:
Worst case timeout is 108min, calculation as follows:

6 eventually loops each timeout 2min: 6 * 2 = 12min
9 sequential subtests, timeout: 9 * 12 = 108min.

Reply:
@Avni-Sharma Should I resolve this conversation?